rotkehlchen/api/v1/resources.py

```diff
@@ -3378,3 +3379,23 @@ def post(self, from_timestamp: Timestamp, to_timestamp: Timestamp) -> Response:
             from_ts=from_timestamp,
             to_ts=to_timestamp,
         )
+
+
+class TimestampHistoricalBalanceResource(BaseMethodView):
+
+    query_schema = SnapshotTimestampQuerySchema()
+    post_schema = HistoricalPerAssetBalanceSchema()
+
+    @require_premium_user(active_check=False)
+    @use_kwargs(query_schema, location='view_args')
+    def get(self, timestamp: Timestamp) -> Response:
```

Comment:
since this also has remote calls it should also be able to be done async. So it's missing the async query argument